# actually T_R_C (C = cam0, R = rgbd cam)
T_B_C:
- [1.0, 0.0, 0.0, 0.0]
- [0.0, 1.0, 0.0, 0.0]
- [0.0, 0.0, 1.0, 0.0]
- [0.0, 0.0, 0.0, 1.0]
invert_T_B_C: false

# actually T_V_C (C = cam0, V = vicon)
T_B_D:
- [0.971048, -0.120915, 0.206023, 0.00114049]
- [0.15701, 0.973037, -0.168959, 0.0450936]
- [-0.180038, 0.196415, 0.96385, 0.0430765]
- [0.0, 0.0, 0.0, 1.0]
invert_T_B_D: true
